### ISSUE TITLE
Slot content doesn't match browser behaviour

### ISSUE THREAD

**Issue by nandi95** [vuejs/vue-test-utils-next#389]
In tests if you use `wrapper.vm.$slots.default()` and you have passed multiple html elements in the default slot, there will be only 1 vnode returned in vtu while in the browser, the correct number of vnodes will get returned.
[Reproduction repo](https://github.com/nandi95/vue-test-utils-next/commit/c4309b40654bc48cade5cfb0d835452f8662998c)

**Comment by xanf** [comment]
@lmiller1990 I'm not sure this one should be a ~bug

Basically the reason for failing is that we're wrapping our slots with SlotWrapper, and this is definitely not the behavior we would want to drop.

As possible action point maybe the text disccused in https://github.com/vuejs/vue-test-utils-next/pull/815#issuecomment-893964846 worth adding not only in changelog, but in docs too, WDYT?

Basically test from reproduction could be fixed in following way:

```js
  it('should return the correct number of vnodes in the slots', () => {
    const wrapper = mount(ComponentWithSlots, {
      slots: {
        default: () => [h('div'), h('div')]
      }
    })

    // @ts-expect-error
    expect(wrapper.vm.$slots.default()).toHaveLength(2)
  })
```

**Comment by xanf** [comment]
Ignore my previous message, I've found a way in #844 to avoid extra wrapper being exposed, so this bug is fixable

**Comment by xanf** [comment]
Closed via #844
